extension/content.js

```diff
@@ -68,6 +68,15 @@ function addReleasesTab() {
 	}
 }
 
+function addTrendingMenuItem() {
+	const $secondButton = $('.header-nav.left .header-nav-item:nth-child(2)');
+	const $trendingButton = $(`<li class="header-nav-item">
```

Comment:
This is a nitpick, but shouldn't it be `$secondListItem` and `$trendingListItem`? They're definitely not `button` elements.